pkg/controller/v1alpha2/applicationconfiguration/render_test.go

```diff
@@ -907,6 +912,79 @@ func TestGetComponent(t *testing.T) {
 	}, c)
 }
 
+var scheme = runtime.NewScheme()
+
+func TestRenderTraitName(t *testing.T) {
```

Comment:
IIRC, this is the case that a trait name exist. It seems that it misses the case that trait doesn't exist

Reply:
done